tests/cloner_test.go

```diff
@@ -0,0 +1,133 @@
+package tests
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/api/core/v1"
+
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	testSuiteName        = "Cloner Test Suite"
+	namespacePrefix      = "cloner"
+	sourcePodFillerName  = "fill-source"
+	sourcePVCName        = "source-pvc"
+	fillData             = "123456789012345678901234567890123456789012345678901234567890"
+	testFile             = utils.DefaultPvcMountPath + "/source.txt"
+	fillCommand          = "echo \"" + fillData + "\" >> " + testFile
+	cloneCompleteTimeout = 10 * time.Second
+)
+
+var _ = Describe(testSuiteName, func() {
+	f, err := framework.NewFramework(namespacePrefix, &framework.Config{})
+	if err != nil {
+		Fail("Unable to create framework struct")
+	}
+
```

Comment:
consider adding a ns var ('var ns string`) and setting this var in the Describe-level BeforeEach, eg.
`ns := Framework.Namespace.Name` 
This saves ~6 references to the field, FWIW, which may not be much...

Reply:
Well the entire block is now its own function, and it doesn't make much sense to put a variable at the top of the block like that (at least I haven't seen anyone else do it).

Reply:
ack